addons/hw_drivers/drivers/DisplayDriver.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, device):
         self.owner = False
         self.rendered_html = ''
         self.load_url()
+        self.call_xdotools('F11')
```

Comment:
put this content in the other commit, as the other commit cannot work properly without this one.

In commit message, explain that as you have removed an extension that was managing the kiosk mode, you have to trigger the key 'F11', that will put the firefox application in full screen mode.

Reply:
All this commit in other one ?
Because we can't call 'F11' without be sure that firefox have a 'windowactivate'.